packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,729 @@
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "While you can declare edge functions inline, you can use this section for more advanced edge function configurations.",
+        "type": "object",
+        "properties": {
+          "cache": {
+            "type": "string",
+            "description": "Should the response of the edge function be cached.",
+            "enum": ["off", "manual"],
+            "x-taplo": {
+              "docs": {
+                "enumValues": [
+                  "Default, edge function response is not cached.",
+                  "Edge function response is cached in the Netlify CDN based on the specified caching headers."
+                ]
+              }
+            }
+          },
+          "function": {
+            "type": "string",
+            "description": "Name of the function."
+          },
+          "path": {
+            "type": "string",
+            "description": "Path pattern to associate with this edge function.",
+            "pattern": "^/.*"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["path", "function"]
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/edge-functions/declarations/"
+        }
+      }
+    },
+    "dev": {
+      "title": "Netlify Dev",
+      "description": "Netlify Dev enables a local development environment without any additional setup and this includes using the functions directory setting to scaffold and serve your functions locally. You can use the `[dev]` section to configure this local development environment. \n\nNote that `[dev]` doesn't run in the Bash shell, so you won't be able to use Bash-compatible syntax with the command.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "command": {
+          "title": "Dev command",
+          "description": "Command that starts your development server or runs a command such as a compiler watch in the background. If no `targetPort` is specified, it runs the command in the background in addition to the static file server.",
+          "type": "string"
+        },
+        "port": {
+          "title": "Port",
+          "description": "Port that Netlify Dev is accessible from in the browser.",
+          "type": "number",
+          "default": 8888
+        },
+        "staticServerPort": {
+          "title": "Static server port",
+          "description": "Port for the static file server. This value will only take effect if `framework` is set to `#static` or `#auto` without any framework being detected.",
```

[dust/sand] small grammar-related edit, hopefully this is still technically correct:

```suggestion
          "description": "Port for the static file server. This value will only take effect if `framework` is set to `#static` or `#auto` and no framework is detected.",
```